qip-0011.md

```diff
@@ -0,0 +1,203 @@
+# Quai-Qi Conversion Controller #
+
+<pre>
+  QIP: 11
+  Layer: Protocol
+  Title: Quai-Qi Conversion Controller
+  Author: kiltsonfire
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/quainetwork/qips/wiki/Comments:QIP-0011
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-02-13
+  License: BSD-2-Clause
+</pre>
+
+## Abstract ##
+
+This QIP defines the reward functions for Qi and Quai as well as the 
+controller which allows protocol conversions between Qi and Quai.
+
+## Motivation ##
+
+Conversions between Quai  and Qi are needed to allow dynamic rebalancing of
```

Comment:
extra space

Reply:
I think maybe it has to do with how you word wrapped the doc? Lots of mis-wrapped paragraphs